From b4903c60d82d06af95a5eb49a73da8865e6ea401 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Mon, 15 Feb 2016 19:08:03 +0900
Subject: [PATCH 1678/2748] HDFS-9765.
 TestBlockScanner#testVolumeIteratorWithCaching
 fails intermittently. (aajisaka)

(cherry picked from commit 8ed07bd3e6b6a8af3b82e9a085c74be29f8dfd87)

Change-Id: I8ffe57ffe92cf6bf234335ca99f9c3c5fe33e53d
---
 .../hdfs/server/datanode/TestBlockScanner.java     |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockScanner.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockScanner.java
index 27fb720..3c766dd 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockScanner.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBlockScanner.java
@@ -245,7 +245,7 @@ public void testVolumeIteratorWithoutCaching() throws Exception {
     testVolumeIteratorImpl(5, 0);
   }
 
-  @Test(timeout=60000)
+  @Test(timeout=300000)
   public void testVolumeIteratorWithCaching() throws Exception {
     testVolumeIteratorImpl(600, 100);
   }
-- 
1.7.9.5

